src/index.html

```diff
@@ -4,9 +4,298 @@
+              <a href="#smartphones" class="nav__link default-link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops&computers"
+                data-qa="nav-hover"
+                class="nav__link default-link"
+              >
+                Laptops &amp Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link default-link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link default-link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#photo" class="nav__link default-link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link default-link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+    <main>
+      <div class="cards">
+        <div class="card cards__card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+          <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+            </div>
+            <div class="card__review-number">Reviews: 5</div>
+          </div>
+          <div class="card__price-wrapper">
+            <div class="card__price-text">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card cards__card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+          <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+              <div class="stars-star"></div>
+            </div>
+            <div class="card__review-number">Reviews: 5</div>
+          </div>
+          <div class="card__price-wrapper">
+            <div class="card__price-text">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a href="#"
```

Comment:
Check indentations. It should match the code style guide and keep the code clean and easily maintainable and readable

Reply:
Should i make the identations between div element? Cause it always was so (previous tasks) and nobody told me to fix that

Reply:
When i wrap the text with tag p instead of div, then tests dont pass and it doesnt work at all. Unfortunately i dont have an  access to our chat cause of my phone so i can't ask it derectly there((